src/components/recordset/recordset-table.tsx

```diff
@@ -120,6 +123,38 @@ const RecordsetTable = ({
 
     rowDetails = tempRowDetails;
   }
+  const [isBottomVisible, setIsBottomVisible] = useState(false);
+  const stickyScrollbarRef = useRef(null);
+  const tableEndRef = useRef(null);
+
+  useEffect(() => {
+    //Only implement intersection observer for top scrollbar when intersectScroll is true otherwise top scrollbar will be sticky all the time
+    if(intersectScroll){
```

Comment:
Instead of having an `if` statement that wraps the whole function, it's cleaner if you check for the opposite and just return:

```tsx
if (!intersectScroll) return;

// the rest of the function goes here 

```

This way you're eliminating an extra indentation (which you didn't even add anyways :D )